charts/hono/templates/hono-service-command-router/hono-service-command-router-deployment.yaml

```diff
@@ -49,7 +49,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        {{- include "hono.component.springEnv" ( merge $args ( dict "additionalProfile" ( ( or .Values.dataGridSpec .Values.dataGridExample.enabled ) | ternary "" "embedded-cache" ) ) ) | indent 8 }}
+        {{- include "hono.component.springEnv" $args | indent 8 }}
```

Comment:
can we also please enable Quarkus based images for the Command Router? This simply requires adding
`"useImageType" true` to the *args* dictionary in the first line of this file ...